src/main/java/ladder/view/InputView.java

```diff
@@ -6,29 +6,23 @@
     private static final Scanner scanner = new Scanner(System.in);
     private static final String INPUT_SEPARATOR = ",";
 
-    private MessagePrinter messagePrinter;
-
-    public InputView() {
-        this.messagePrinter = new MessagePrinter();
-    }
-
     public String[] inputPersonNames() {
-        messagePrinter.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
+        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
```

Comment:
```suggestion
        System.out.printf("%n실행 결과를 입력하세요. (결과는 쉼표(,)로 구분하세요)%n");
```
intellij 를 사용하신다면 코드 개선에 대한 지원을 받으실 수 있습니다. 😀 
![Capture 2021-04-15 오후 9 19 31](https://user-images.githubusercontent.com/247208/114867900-4816ba80-9e30-11eb-8119-15eddb7c78e1.jpg)